packages/office-ui-fabric-react/src/utilities/dateValues/TimeConstants.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Common constants and methods
+ */
+let TimeConstants = {
+  MillisecondsInOneDay: 86400000,
+  MillisecondsIn1Sec: 1000,
+  MillisecondsIn1Min: 60000,
+  MillisecondsIn30Mins: 1800000,
+  MillisecondsIn1Hour: 3600000,
+  MinutesInOneDay: 1440,
+  MinutesInOneHour: 60,
+  DaysInOneWeek: 7,
+  MonthInOneYear: 12,
+  WeeksInOneYear: 52
```

Comment:
Expose only the things you need, are you using all these? Also, OWA JSMVVM shows 53 weeks for this year:
![image](https://user-images.githubusercontent.com/11838969/31194756-fd63727a-a8fc-11e7-8eae-387de62e65ff.png)

Reply:
@johannao76 what do you have set at firstWeekOFYear?